src/Makefile.am

```diff
@@ -196,7 +196,9 @@ gen_internal_compose_table_SOURCES = \
     ibuskeyuni.c        \
     ibusenumtypes.h     \
     $(NULL)
-gen_internal_compose_table_CFLAGS = $(AM_CFLAGS)
+gen_internal_compose_table_CFLAGS = \
+	$(AM_CFLAGS) \
+	-DIBUS_BUILDING_GEN_INTERNAL_COMPOSE_TABLE
```

Comment:
I wish to delete ibus.h in src/gencomposetable.c instead of this change.

Reply:
Thanks, that seems to work. I'll make that change.